show the hostname before being resolved, in case it fails eg. during youtube-dl (without --verbose)

--- a/Lib/socket.py	2017-08-16 21:37:26.912201461 +0200
+++ b/Lib/socket.py	2017-08-16 21:31:05.805992615 +0200
@@ -740,6 +740,7 @@ def getaddrinfo(host, port, family=0, ty
     # We override this function since we want to translate the numeric family
     # and socket type values to enum constants.
     addrlist = []
+    print("!! host=",host,file=sys.stderr)
     for res in _socket.getaddrinfo(host, port, family, type, proto, flags):
         af, socktype, proto, canonname, sa = res
         addrlist.append((_intenum_converter(af, AddressFamily),
